podpac/core/interpolation/interpolators.py

```diff
@@ -270,25 +270,16 @@ def interpolate(self, udims, source_coordinates, source_data, eval_coordinates,
             )
 
         def get_rasterio_transform(c):
-            """Summary
-            
-            Parameters
-            ----------
-            c : TYPE
-                Description
-            
-            Returns
-            -------
-            TYPE
-                Description
-            """
-            west, east = c["lon"].area_bounds
-            south, north = c["lat"].area_bounds
-            cols, rows = (c["lon"].size, c["lat"].size)
-            # print (east, west, south, north)
+            west = c["lon"].bounds[0] - c["lon"].step / 2.0
+            east = c["lon"].bounds[1] + c["lon"].step / 2.0
+            south = c["lat"].bounds[0] - c["lat"].step / 2.0
+            north = c["lat"].bounds[1] + c["lat"].step / 2.0
+            cols = c["lon"].size
+            rows = c["lat"].size
```

Comment:
@mpu-creare This is what we would replace `c.geotransform` if possible, but I couldn't figure out how exactly.

Reply:
This is fine. As a one-liner, it would have been 
```python
return rasterio.Affine.from_gdal(*c.geotransform)
```
But this is pretty clear in case of future debugging...